clang/test/Driver/linker-wrapper-sycl-win.cpp

```diff
@@ -116,3 +116,17 @@
 // Error handling when --linker-path is not provided for clang-linker-wrapper
 // RUN: not clang-linker-wrapper 2>&1 | FileCheck --check-prefix=LINKER-PATH-NOT-PROVIDED %s
 // LINKER-PATH-NOT-PROVIDED: linker path missing, must pass 'linker-path'
+
+/// Check --device-lib-dir for sycl-post-link tool
+// ------
+// Generate .o file as linker wrapper input.
+
+// RUN: %clang %s -fsycl -fsycl-targets=spir64-unknown-unknown -c --offload-new-driver -o %t5.o
+//
+// Run clang-linker-wrapper test
+//
+// RUN: clang-linker-wrapper -sycl-post-link-options="SYCL_POST_LINK_OPTIONS" -llvm-spirv-options="LLVM_SPIRV_OPTIONS" "--host-triple=x86_64-pc-windows-msvc" "--linker-path=/usr/bin/ld" "--" HOST_LINKER_FLAGS "-dynamic-linker" HOST_DYN_LIB "-o" "a.out" HOST_LIB_PATH HOST_STAT_LIB %t5.o -sycl-device-libraries=libsycl-crt.new.o -sycl-device-library-location=%S/Inputs/SYCL/lib --dry-run 2>&1 | FileCheck -check-prefix=CHK-CMDS-DEVICE-LIB-DIR %s
```

Comment:
Do we need this extra file? Can we not just use a dummy file? Please see https://github.com/intel/llvm/blob/b512633e2de8547cecfce12aed2c62090282c6ed/clang/test/Driver/linker-wrapper-sycl.cpp#L61
If you do not care about what's inside the file, then `touch %t1.devicelib.o` will suffice.

Reply:
Hi, @asudarsa 
We need to pass "-sycl-device-library-location" to clang-linker-wrapper tool and clang-linker-wrapper will look for specified devicelib files in this specified "-sycl-device-library-location", so we put an empty devicelib file in Inputs/SYCL/lib
Thanks very much.